workers/social/lib/social/models/user/index.coffee

```diff
@@ -692,15 +693,17 @@ module.exports = class JUser extends jraphical.Module
 
         user.fetchAccount context, (err, account) ->
 
-          if err?
+          if err
 
             logout 'error fetching account', clientId, callback
 
           else
-
             # A valid session, a valid user attached to
             # it voila, scenario #2
-            callback null, { session, account }
+            session.lastAccess = lastAccess = new Date
```

Comment:
Right it's too much for now, I'll disable it as you suggested but eventually we need to move session to Redis or some other in memory first db, we can implement the last activity updater later.